security/compliance_operator/compliance-operator-troubleshooting.adoc

```diff
@@ -0,0 +1,27 @@
+[id="compliance-operator-troubleshooting"]
+= Troubleshooting the Compliance Operator
+include::modules/common-attributes.adoc[]
+:context: compliance-troubleshooting
+
+toc::[]
+
+This section describes how to troubleshoot the Compliance Operator. The information can be useful either to diagnose a problem or provide information in a bug report. Some general tips:
+
+* The Compliance Operator emits Kubernetes events when something important happens. You can either view all events in the cluster using the command `oc get events -nopenshift-compliance` or events for an object, such as a scan using the command `oc describe compliancescan/$scan`.
+
+* The Compliance Operator consists of several controllers, approximately one per API object. It could be useful to filter only those controllers that correspond to the API object having issues. If a `ComplianceRemediation` can't be applied, view the messages from the `remediationctrl` controller. You can filter the messages from a single controller e.g. using `jq`: `oc logs compliance-operator-775d7bddbd-gj58f | jq -c 'select(.logger == "profilebundlectrl")'`
```

Comment:
the command `oc logs compliance-operator-775d7bddbd-gj58f | jq -c 'select(.logger == "profilebundlectrl")' didn't show full command in https://new-osdocs-1466--ocpdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-troubleshooting.html.
Could you update it to command format? Thanks.